src/main/java/com/notification/service/impl/NotificationServiceImpl.java

```diff
@@ -176,4 +181,38 @@ public InvitationDetails sendHotelEmail(InviteRequest inviteRequest) {
 
 		return savedInvitation;
 	}
+
+	@Override
+	public ResponseModel<String> sendPasswordResetLinkEmailToUser(PasswordResetTokenRequest passwordResetTokenRequest) {
+//		boolean emailExists = passwordResetTokenDetailsRepository.existsByUserEmail(passwordResetTokenRequest.getUser().getEmail());
+//		if (emailExists){
+//			throw new BadRequestException("This email " + passwordResetTokenRequest.getUser().getEmail() + " already exists in our system.");
+//		}
+		String fullName = passwordResetTokenRequest.getUser().getFirstName() + " " + passwordResetTokenRequest.getUser().getLastName();
+		String passwordResetLink = PASSWORD_RESET_LINK + passwordResetTokenRequest.getToken();
+
+
+		PasswordResetTokenDetails passwordResetTokenDetails = new PasswordResetTokenDetails();
+		passwordResetTokenDetails.setUserEmail(passwordResetTokenRequest.getUser().getEmail());
+		passwordResetTokenDetails.setToken(passwordResetTokenRequest.getToken());
+		passwordResetTokenDetails.setPasswordResetLink(passwordResetLink);
+
+		ResponseModel<String> response = new ResponseModel<>();
+		try {
+			passwordResetTokenDetailsRepository.save(passwordResetTokenDetails);
+			Context thymeleafContext = new Context();
+			thymeleafContext.setVariable("sentToName", fullName);
```

Comment:
sentToName literals should be final constant variable

Reply:
Resolved